verifier/src/report_body.rs

```diff
@@ -219,13 +231,57 @@ impl<T: Accessor<MiscellaneousSelect>> Verifier<T> for MiscellaneousSelectVerifi
     }
 }
 
+/// Verify the [`MrEnclave`] is as expected.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct MrEnclaveVerifier {
+    expected_measurement: MrEnclave,
+}
+
+impl MrEnclaveVerifier {
+    /// Create a new [`MrEnclaveVerifier`] instance.
+    ///
+    /// # Arguments:
+    /// * `expected_measurement` - The expected measurement.
+    pub fn new(expected_measurement: MrEnclave) -> Self {
+        Self {
+            expected_measurement,
+        }
+    }
+}
+
+impl Accessor<MrEnclave> for ReportBody {
+    fn get(&self) -> MrEnclave {
+        let Measurement::MrEnclave(mr_enclave) = self.mr_enclave() else {
+            panic!("`mr_enclave()` should return a Measurement::MrEnclave");
+        };
+        mr_enclave
+    }
+}
```

Comment:
This is here until, https://github.com/mobilecoinfoundation/sgx/issues/286.
Added as a task to #7